src/status_im2/subs/root.cljs

```diff
@@ -97,6 +97,7 @@
 (reg-root-key-sub :group-chat-profile/editing? :group-chat-profile/editing?)
 (reg-root-key-sub :group-chat-profile/profile :group-chat-profile/profile)
 (reg-root-key-sub :selected-participants :selected-participants)
+(reg-root-key-sub :deselected-members :deselected-members)
```

Comment:
As I mentioned earlier in this PR, we should generally avoid root state like this due to performance issues, but also because it's difficult to reason about it. For instance, "deselected members of what?".

Unfortunately the current app DB is badly designed in many ways, so I understand you're kind of just going with the grain.

A better design IMO would group related subscriptions under a common bounded context (e.g. chat or group chat, etc). This is really important, perhaps we should document it somehow.